pkg/utils/yaml_func_env.go

```
@@ -0,0 +1,69 @@
+package utils
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/charmbracelet/log"
```

golangci-lint (1.64.8)

[error] import "github.com/charmbracelet/log" imported without alias but must be with alias "log" according to config

(importas)